### ISSUE TITLE
retrieveMessageContent is not returning the correct buffer

### ISSUE THREAD

**Issue by mybaseball52** [Yoctol/messaging-apis#479]
I have found that if I pass any different picture, the buffer is always return

`<Buffer ff d8 ff e0 00 10 4a 46 49 46 00 01 01 01 00 01 00 01 00 00 ff db 00 43 00 03 02 02 03 02 02 03 03 03 03 04 03 03 04 05 08 05 05 04 04 05 0a 07 07 06 ... >`

So I have try use node fetch to fix this problem for my project.

**Comment by wtflink** [comment]
Hi @mybaseball52, if we set everything up fairly, there shouldn't be any difference between using node fetch or axios. If you can provide more info such as the whole buffer you keep receiving, it would be very helpful for us to target down the real problem here.

**Comment by chentsulin** [comment]
Hi @mybaseball52, I finally found that this is a bug we introduced in this commit and it misses the `responseType: 'arraybuffer'` setting:
https://github.com/Yoctol/messaging-apis/commit/a4a0479f784ddfbc6fa48f4e86365be00e35f5a0#diff-678a1f28cbda070ea9f76ab2e5aadf6aR568

I'm going to add a fix and release a v0.8.3 version. Thanks for your information.

BTW, when we do `console.log` to the buffer, it only shows some bytes that usually represent the header of the file, so you can see the same `<Buffer xxx ...>` output but files are actually different.

```
<Buffer ff d8 ff e0 00 10 4a 46 49 46 00 01 01 01 00 01 00 01 00 00 ff db 00 43 00 03 02 02 03 02 02 03 03 03 03 04 03 03 04 05 08 05 05 04 04 05 0a 07 07 06 ... >
```

**Comment by chentsulin** [comment]
@mybaseball52, v0.8.3 has been released. You can try it out:

https://github.com/Yoctol/messaging-apis/releases/tag/v0.8.3
